examples/xfixes-cursor-notify.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+#
+# examples/xfixes-cursor-notify.py -- demonstrate the XFIXES extension
+# CursorNotify event.
+#
+#    Copyright (C) 2022
+#      Dan Isla <dan.isla@gmail.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation; either version 2.1
+# of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the
+#    Free Software Foundation, Inc.,
+#    59 Temple Place,
+#    Suite 330,
+#    Boston, MA 02111-1307 USA
+
+# Python 2/3 compatibility.
+from __future__ import print_function
```

Comment:
It's better to say the example is for Python 3.x only, because `print("...", file=sys.stderr)` is not compatible with Py2.7 (lines 38 and 45). Or you can remove `file=sys.stderr` parameter for Py2.7 compatibility which is not dropped yet for the library itself.

Reply:
sounds good, I'll just remove the file= arg since it's just an example.